General/Baselines.md

```diff
@@ -164,11 +164,11 @@ This snippet should be configured with:
   - `bitbucket` for bitbucket
   - `gitlab` for gitlab
   - `git` for others (and {url} is thus mandatory)
-- `{url}`: base url to the git host. Mandatory when prefix `git` is used.
+- `{url}`: Base url to the git host. Mandatory when prefix `git` is used, not needed for other prefixes
```

Comment:
This is not true.
It can be use for a dependency to a self-hosted GitLab instance